pastepwn/util/tests/templatingengine_test.py

```diff
@@ -33,8 +33,37 @@ def setUp(self):
                            p.get("full_url"))
 
     def test_fill_template(self):
+        """Checks if templating engine inserts paste data correctly into the template"""
         analyzer_name = "TestAnalyzer"
         template = "Matched paste '${key}' by analyzer '${analyzer_name}'. URL is: '${full_url}'"
-        comparison = "Matched paste '{0}' by analyzer '{1}'. URL is: '{2}'".format(self.p.get("key"), analyzer_name, self.p.get("full_url"))
+        expected = "Matched paste '{0}' by analyzer '{1}'. URL is: '{2}'".format(self.p.get("key"), analyzer_name, self.p.get("full_url"))
         result = TemplatingEngine.fill_template(paste=self.paste, analyzer_name=analyzer_name, template_string=template)
-        self.assertEqual(comparison, result, msg="Filled template string is not the same as the expected result!")
+        self.assertEqual(expected, result, msg="Filled template string is not the same as the expected result!")
+
+    def test_fill_template_matches(self):
+        """Checks if templating engine inserts the matches correctly into the template"""
+        template = "Matches are: ${matches}"
+        expected = "Matches are: +123456789\n+987654321\n".format("")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Too many arguments for format string](https://app.codacy.com/manual/d-Rickyy-b/pastepwn/pullRequest?prid=4945865)